scv-web/.gitignore

```diff
@@ -0,0 +1,21 @@
+.DS_Store
```

Comment:
Move `.gitignore` file to the top level, so we just have one for the whole project.